orchestrator-salt/src/main/resources/salt/salt/nginx/conf/clouderamanager-ssl-user-facing.conf

```diff
@@ -28,6 +28,14 @@ server {
         rewrite ^([/](?!clouderamanager/).*$) /clouderamanager$1;
     }
 
+    # If hadoop-jwt cookie not set and favicon is requested return 403.
+    if ($cookie_hadoop-jwt !~ .+) {
```

Comment:
I suspect this rule will be applied when CM is reached directly and not through Knox.
Could you please test this behaviour when NGiNX proxies the request directly to CM?
https://github.com/hortonworks/cloudbreak#how-to-reach-cm-ui-directlynot-through-knox 

The mentioned scenario is mainly for debugging when the cluster could not be installed and may Knox is not running.
Otherwise the PR looks good to me.

Reply:
@biharitomi thanks! looks like this is not affecting direct CM requests. This is a direct call to CM, notice there is no response from the server (not even 403). If this was going through the patched nginx code we would have seen 403. 
![image](https://user-images.githubusercontent.com/380997/86352886-81c95c80-bc34-11ea-81d9-c455125af95b.png)

When the request goes through Knox we return 403 (nginx patch code)
![image](https://user-images.githubusercontent.com/380997/86353255-f56b6980-bc34-11ea-8c20-9eaf60e276cf.png)

Which makes me conclude this patch works as expected. Let me know your thoughts.

Reply:
Thanks!